docs/guides/high-availability/single-cluster/deploy-keycloak.adoc

```diff
@@ -0,0 +1,94 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Deploying {project_name} across multiple availability-zones with the Operator"
+summary="Deploy {project_name} for high availability with the {project_name} Operator as a building block."
+tileVisible="false" >
+
+This guide describes advanced {project_name} configurations for {kubernetes} which are load tested and will recover availability-zone
+failures.
+
+These instructions are intended for use with the setup described in the <@links.ha id="single-cluster-concepts"/> {section}.
+Use it together with the other building blocks outlined in the <@links.ha id="single-cluster-building-blocks"/> {section}.
+
+[#single-cluster-deploy-keycloak-prerequisites]
+== Prerequisites
+
+* {kubernetes} cluster deployed across multiple availability-zones with a worker-pool configured for each.
+* Understanding of a <@links.operator id="basic-deployment" /> of {project_name} with the {project_name} Operator.
+* AWS Aurora database deployed using the <@links.ha id="single-cluster-deploy-aurora" /> {section}.
+
+[#single-cluster-deploy-keycloak-procedure]
+== Procedure
+
+. Determine the sizing of the deployment using the <@links.ha id="single-cluster-concepts-memory-and-cpu-sizing" /> {section}.
+
+. Install the {project_name} Operator as described in the <@links.operator id="installation" /> {section}.
+
+. Notice the configuration file below contains options relevant for connecting to the Aurora database from <@links.ha id="single-cluster-deploy-aurora" anchor="single-cluster-aurora-connecting" />
```

Comment:
Double check if the `anchor="single-cluster-aurora-connecting"` exists. The link seems broken, but I haven't built the website to double check.